go-controller/pkg/ovn/udn_isolation.go

```diff
@@ -240,29 +240,28 @@ func (oc *DefaultNetworkController) getUDNOpenPortDbIDs(podNamespacedName string
 		})
 }
 
-const advertisedNetworkIsolationACLID = "advertised-network-isolation"
-const advertisedNetworkSubnetsAddressSet = "advertised-network-subnets"
-const advertisedNetworkSubnetsCtrl = "advertised-network-subnets-controller"
+// advertisedNetworkSubnetsKey is the object name key for the global advertised networks addressset and the global deny ACL
+const advertisedNetworkSubnetsKey = "advertised-network-subnets"
 
 // GetAdvertisedNetworkSubnetsAddressSetDBIDs returns the DB IDs for the advertised network subnets addressset
 func GetAdvertisedNetworkSubnetsAddressSetDBIDs() *libovsdbops.DbObjectIDs {
-	return libovsdbops.NewDbObjectIDs(libovsdbops.AddressSetAdvertisedNetwork, advertisedNetworkSubnetsCtrl, map[libovsdbops.ExternalIDKey]string{
-		libovsdbops.ObjectNameKey: advertisedNetworkSubnetsAddressSet,
+	return libovsdbops.NewDbObjectIDs(libovsdbops.AddressSetAdvertisedNetwork, DefaultNetworkControllerName, map[libovsdbops.ExternalIDKey]string{
+		libovsdbops.ObjectNameKey: advertisedNetworkSubnetsKey,
 	})
 }
 
 // GetAdvertisedNetworkSubnetsDropACLdbIDs returns the DB IDs for the advertised network subnets drop ACL
 func GetAdvertisedNetworkSubnetsDropACLdbIDs() *libovsdbops.DbObjectIDs {
-	return libovsdbops.NewDbObjectIDs(libovsdbops.ACLAdvertisedNetwork, advertisedNetworkIsolationACLID,
```

Comment:
oh yea I did think it was weird there was a controller name added like that instead of using the default one.. but now looks better!